refresh/index.js

```diff
@@ -655,6 +685,58 @@ module.exports = generators.Base.extend({
       });
   },
 
+  generateSDKs: function() {
+    if(!this.fromSwagger && this.serverSwaggerFiles <= 0) return;
```

Comment:
`this.serverSwaggerFiles <= 0` looks like it should be `this.serverSwaggerFiles.length <= 0`.